master/buildbot/statistics/capture.py

```diff
@@ -118,32 +167,36 @@ def consume(self, routingKey, msg):
         Consumer for CaptureBuildStartTime. Gets the build start time.
         """
         builder_info = yield self.master.data.get(("builders", msg['builderid']))
-        if self._builder_name == builder_info['name']:
+        if self._builder_name_matches(builder_info):
             try:
                 ret_val = self._callback(*self._retValParams(msg))
             except Exception as e:
                 # catching generic exceptions is okay here since we propagate it
-                config.error(self._err_msg(msg) + " Exception raised: " + type(e).__name__ +
+                config.error(self._err_msg(msg, builder_info['name']) + " Exception raised: " + type(e).__name__ +
                              " with message: " + str(e))
-            context = self._defaultContext(msg)
+            context = self._defaultContext(msg, builder_info['name'])
             post_data = {
                 self._time_type: ret_val
             }
-            series_name = self._builder_name + "-build-times"
+            series_name = builder_info['name'] + "-build-times"
```

Comment:
I should've commented on this earlier: '%s-build-times' % (builder_info['name'],)` would be better.